src/tests/unit/test_check_existence.py

```diff
@@ -0,0 +1,40 @@
+import unittest
+from copy import deepcopy
+from unittest import mock
+
+from common.address import Address
+from features.document.use_cases.check_exsistence_use_case import (
+    check_existence_use_case,
+)
+from tests.unit.mock_utils import get_mock_document_service
+
+
+class CheckExistenceTestCase(unittest.TestCase):
+    def setUp(self) -> None:
+        self.storage = {
+            "1": {
+                "name": "Test document",
```

Comment:
```suggestion
                "name": "Test_document",
```
We have defined a rule that name should not contain spaces (see EntityName class in shared.py). Not sure if we have added validation for that...